src/middlewares/index.js

```diff
@@ -1,3 +1,17 @@
+const cors = require('cors');
+
+const allowedOrigins = ['https://web.coding.garden', 'https://web-dev.coding.garden'];
+
+const corsOptions = {
+  origin: (origin, callback) => {
```

Comment:
Can you use a shorthand method name here?

Reply:
Like this: ?
```javascript
const corsOptions = {
  origin(origin, callback) {
    if (process.env.NODE_ENV === 'development' || allowedOrigins.includes(origin)) {
      callback(null, true);
    } else {
      callback(new Error('Not allowed by CORS'));
    }
  },
};
```

Reply:
Yeah, does this cause a linting error though? Since you have 2 `origin`s in the same scope? Should one of them be renamed?

Reply:
No errors, should be fine.

Reply:
Ok, last question. Does Now set the env to production? I just want to make sure this will definitely open up the dev deployment.

Reply:
We set the NODE_ENV when deploying depending on which branch is pushed to. I’m not sure if Now ignores this and starts the app in production. I can test this tomorrow, unless anyone else can confirm or deny this.

Reply:
@sbibow Can you give some input here? Any ideas?

Reply:
I'm pretty sure it's not in production, fetching this URL `https://api-dev.coding.garden/123` return an error with a stack trace, this should only happen if the app is not in production.

Reply:
That is a good test. Merging now.